### DISCUSSION TITLE
Additional ways to Limit Exploit Range of Conveyor Systems

### DISCUSSION THREAD

**Discussion by JezzyBelle** [voxeltycoon/issues#389] [General]
It is too easy to create a large, map-wide, conveyor ONLY, logistical system. I suggest that there needs to be more factors added to help limit its exploitation. When I found out you could exploit them that way it sorta bummed me out!... lol

So, throwing some ideas out there.

1. Currently. There is a limit on contiguous length you can lay. That's a start.

2. Dead Zones. Limit its use across and into newly purchased areas by creating a dead zone for building anything other than, currently, roads and track. There may be other things but nothing that can can allow you to connect or out down a conveyor. 

3. Make Them Pay. Make it more expensive the longer the conveyor. Current max length is 375m. On level ground it is currently $2370 for 5m. At max length of 375m it comes to 175,140. If you were to price it at the 5m pricing for each 5m in length that 374m would cost vast amounts more at 4,449,750. It almost seems to encourage you to exploit.

4. Speed Limit. Speed of conveyor should be slower than trucks anyway so should be adjusted. BUT, as the length is longer, make it go slower. 

5. Some Shall Pass. Couple of things. Maybe some items, much like passengers, should not be conveyable. Simple. Also, some items should require special conveyors. Easy one would be liquids in the form of pipelines. I could see maybe a sluice for logs. Some could only be shorter distances and have to be enclosed for more finished products. Make them insanely expensive and shorter max distance

**Comment by hairintd** [comment]
one other way - longer conveyors should be harder to build for player. Like part length could be limited greatly and add necessary to build a connector tower (belt engine or whatever) between parts for example..
